client/app/app.js

```diff
@@ -70,6 +70,10 @@ angular.module("app", [
 .run(AppLocalizeRun)
 .config(AppMaterial)
 .config(AppHTTPErrorHandler)
+.config(( $mdGestureProvider ) => {
```

Comment:
I'm not quite sure if tinkering with the mdGestureProvider is a good idea - but otherwise clicks are not working on mobile maps.